src/value.ml

```diff
@@ -1,18 +1,46 @@
-open Stdune
+open Import
 
 type t =
   | String of string
   | Dir of Path.t
   | Path of Path.t
 
+let sexp_of_t =
+  let open Sexp.To_sexp in
+  function
+  | String s -> (pair string string) ("string", s)
+  | Path p -> (pair string Path.sexp_of_t) ("path", p)
+  | Dir p -> (pair string Path.sexp_of_t) ("dir", p)
+
 let string_of_path ~dir p = Path.reach ~from:dir p
 
+let compare x y =
+  match x, y with
+  | String x, String y -> String.compare x y
+  | Path x, Path y -> Path.compare x y
```

Comment:
The `Dir x, Dir y` case is missing

Reply:
Actually, we don't need this function anyway. I got rid of it.